File: blog_springboot/src/main/java/com/nap_blog/entity/Visits.java
package com.nap_blog.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Visits {
    @TableId(type = IdType.AUTO)
    private Long id;
    private String ip;
    private String browser;
    private String city;
    private String fromUrl;
    private Date visitTime;
}

File: blog_admin/types/Visits.ts
import { type PageQuery } from '~/types/PageQuery'
export interface Visits {
    id?: number,
    ip: string,
    city: string,
    fromUrl?: string,
    browser?: string,
    visitTime?: string
}
export interface VisitsQuery extends PageQuery {
    browser?: string
    ip?: string,
    city?: string,
    visitTime?: string,
    fromUrl?: string
} 